tests/jballerina-unit-test/src/test/resources/test-src/enums/enums.bal

```diff
@@ -48,6 +51,9 @@ function testBasicEnumSupport() {
 
     string q = QUEEN;
     assert(q, "QUEEN");
+
+    string b = BEATLES;
```

Comment:
Shall we add a test where `BEATLES` is used as a type as well?
e.g., 
```ballerina
BEATLES btl = "The Beatles";
```

Reply:
Sure. Added.